app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -17,18 +17,10 @@ def create
         order = Order.find(payment_intent.transfer_group)
 
         return if order.paid?
+        latest_charge = Stripe::Charge.retrieve(payment_intent.latest_charge, api_key: marketplace.stripe_api_key)
```

Comment:
Nit: a blank line after the guard clause would be nice

Reply:
Good call!